File: set_1/mario.c
#include<stdio.h>
#include<cs50.h>

int get_height(void);

//Generates a pyramid of height n.
int main(void)
{
    int n = get_height();
    for (int i = 0; i < n; i++)
    {
        //This prints n spaces minus number of rows.
        for (int j = 0; j < n - i - 1; j++)
        {
            printf(" ");
        }
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        //The gap between.
        printf("  ");
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

//Gets an integer between 1 and 8 from the user to be the height of the pyramid.
int get_height(void)
{
    int n;
    do
    {
        n = get_int("Height:");
    }
    while (n < 1 || n > 8);
    return n;
}

File: set_6/mario.py
from cs50 import get_int

while True:
    h = get_int("Height: ")     # get height from user thanks to get_int
    if h > 0 and h < 9:     # check height between 1 and 8
        break

for i in range(h):
    print(" " * (h - 1 - i) + "#" * (i + 1) + "  " + "#" * (i + 1))     # print the two pyramids